pandas/core/sorting.py

```diff
@@ -47,10 +50,10 @@
 
 def get_indexer_indexer(
     target: Index,
-    level: str | int | list[str] | list[int],
+    level: Level | list[Level] | None,
     ascending: Sequence[bool | int] | bool | int,
-    kind: str,
-    na_position: str,
+    kind: SortKind,
```

Comment:
I think using `int` here is wrong, based on the docs for this function.

Reply:
removed int